test/integration/transactions_test.go

```diff
@@ -26,7 +27,7 @@ import (
 // const privateKey = "451EA3199FE0520FB10B7F89D3A34BAF7E5C3B16FDFE2BC11A5CAC95CDB29ED6"
 
 const testUrl = "http://127.0.0.1:3000"
-const privateKey = "28FCECEA252231D2C86E1BCF7DD541552BDBBEFBB09324758B3AC199B4AA7B78"
+const privateKey = "EDFB348D4AAA333E6D73D9CAD1EA18FE3FE079CC3373E9E4E75A4FBD7D3476E0"
```

Comment:
don't push changed `privateKey` =)

Reply:
Removed in https://github.com/proximax-storage/go-xpx-chain-sdk/pull/105/commits/a0b74d8e3cba5f9c5e9c0d7e45d15e1a4613d178